core/blockchain.go

```diff
@@ -1988,6 +1991,10 @@ func (bc *BlockChain) processBlock(block *types.Block, statedb *state.StateDB, s
 	blockWriteTimer.Update(time.Since(wstart) - max(statedb.AccountCommits, statedb.StorageCommits) /* concurrent */ - statedb.SnapshotCommits - statedb.TrieDBCommits)
 	blockInsertTimer.UpdateSince(start)
 
+	blockGasUsedGauge.Update(int64(usedGas))
+	blockBaseFeeGauge.Update(block.Header().BaseFee.Int64())
```

Comment:
Looks like tests are failing here.

Reply:
Thanks for the heads ups. I wonder what can be the situation where basefee is nil 🤔

Reply:
Pre-EIP-1559 blocks shouldn't contain it as well, no?